.gitignore

```
@@ -11,7 +11,7 @@ __pycache__/
 
 # Test files and directories
 .pytest_cache/
-.coverage
+.coverage-*
```

I would suggest this so that existing `.coverage` files (or `.coverage` files in projects that don't have the new workflow yet) will continue to be ignored.

```suggestion
.coverage*
```